lib/generators/graphql/templates/base_connection.erb

```diff
@@ -4,5 +4,9 @@ module Types
     # add `nodes` and `pageInfo` fields, as well as `edge_type(...)` and `node_nullable(...)` overrides
     include GraphQL::Types::Relay::ConnectionBehaviors
   end
+
+  class BaseObject
```

Comment:
I'm surprised that reopening this class would work without re-declaring its super class, for example: 

```suggestion
  class BaseObject < GraphQL::Schema::Object 
```

But, is this code working for you in your project?

Reply:
The code is working in my project. I couldn't find any clear documentation on whether or not you must specify the subclass when re-opening a class: my experiments show that it work. Consider the following code:

```
class A
  def hi
    puts "hi"
  end
end

class B < A
  def bye
    puts "bye"
  end
end

class B
  def aloha
    puts "aloha"
  end
end


b = B.new

b.hi
b.bye
b.aloha
puts b.is_a?(A)
```

Output:
```
hi
bye
aloha
true
```